### ISSUE TITLE
Preset applied is no longer displayed in bold text

### ISSUE THREAD

**Issue by chhil** [darktable-org/darktable#7806]
Current master.
WSL Ubuntu.

Apply a preset.
Go back into the preset menu to check which one you applied. It's no longer shown in bold.

**Comment by elstoc** [comment]
We need much more information than this to investigate a fault. Please refer to the template that is shown when you create a new bug.

**Comment by chhil** [comment]
this is darktable 3.5.0+557~g713fac470

compile options:
  bit depth is 64 bit
  normal build
  SSE2 optimized codepath enabled
  OpenMP support enabled
  OpenCL support enabled
  Lua support enabled, API version 6.2.0
  Colord support enabled
  gPhoto2 support enabled
  GraphicsMagick support enabled
  ImageMagick support disabled
  OpenEXR support enabled

A preset has been applied but it's no longer in bold.

![image](https://user-images.githubusercontent.com/1119439/104303866-a14d1600-54f0-11eb-8bd1-8dab86d0defb.png)

**Comment by elstoc** [comment]
I've tested this in 3.5.0+558~gd82d67fbe (ArchLinux) and I can't reproduce. I've tried exiting and re-entering the darkroom and restarting the application and the selected preset still remains bold as expected.

**Comment by chhil** [comment]
You have no custom css in preferences , its at default?

**Comment by elstoc** [comment]
I did have some custom CSS but nothing that would impact the bold setting. Anyway I still can't reproduce the issue if I disable my customisations.

**Comment by TurboGit** [comment]
Select it, then update it. I bet there was some changes into the structure that makes the match fails. Can you confirm this works on your side ? I kind of remember having seen/done that.

**Comment by chhil** [comment]
I did a clean build and install 3.5.0+558~gd82d67fbe, still doesn't work for me.

**Comment by chhil** [comment]
@TurboGit 
All presets don't get bold.

I updated a presets value and updated the preset, still no change.

**Comment by chhil** [comment]
A recent install did a db upgrade, could it be related?

**Comment by TurboGit** [comment]
Maybe related to the fix to bold on Windows ?

Can you try reverting df7d40f45165a82c258696aabed7513f8995bcc4 ?

**Comment by chhil** [comment]
I am not on windows (WSL ubuntu) but I can try.

Any easy command to revert to a release before that commit?

**Comment by chhil** [comment]
BTW Lighttable preset are showing up bold. The problem is in darkroom module presets.

**Comment by Nilvus** [comment]
Are you sure you not change, even a little, a setting in the module? Preset selected remains selected if all settings of the module are the preset ones. Just a little change unselect it and it's normal. Windows fix didn't change that. I have no issue at all in actual master and even since that fix in Ubuntu. And I've discussed and tested Windows fix by @Mark-64 on my side. So something probably on your side here. And I of course talk of darkroom ones.

**Comment by chhil** [comment]
> Are you sure you not change, even a little, a setting in the module?

Yes

![NoBoldPreset](https://user-images.githubusercontent.com/1119439/104323233-f0ed0b00-550b-11eb-8452-d9ed3d0b6ebf.gif)

**Comment by Mark-64** [comment]
With the commit previously mentioned the control of bold fontface has been moved to css, so you need to use the new css themes. So please make sure you are not using an old custom css theme form your local config forder under /themes

**Comment by Mark-64** [comment]
... or maybe you don't have installed any of the fonts f the fallback list

```
.active-menu-item * /* needed for some Pango issues not rendering synthetic bold for all OS */
{
    font-family: "Roboto",
                 "Segoe UI",
                 "SF Pro Text",
                 "Ubuntu",
                 "Cantarell",
                  sans-serif;
}
```
Can you check ?
BTW are you using darktable-elegant-darker or the default darktable theme?

**Comment by Nilvus** [comment]
To complete @Mark-64: you could also show us your CSS/theme settings. That works great on our side.

And also, do you delete completely destination install folder of darktable AND build folder in folder where you compile darktable, before any new compilation?

**Comment by chhil** [comment]
> With the commit previously mentioned the control of bold fontface has been moved to css, so you need to use the new css themes. Please make sure you are not using an old custom css theme from your local config forder under /themes

I deleted the build directory and install directory. Built dt, it's a clean dt install, so whatever came with the build is present.

>or maybe you don't have installed any of the fonts f the fallback list

I run wsl ubuntu on windows 10.

The available fonts...
```
mchhil@DESKTOP-MKCA8HN:~/darktable$ fc-list
/usr/share/fonts/truetype/dejavu/DejaVuSerif-Bold.ttf: DejaVu Serif:style=Bold
/usr/share/fonts/opentype/urw-base35/URWBookman-LightItalic.otf: URW Bookman:style=Light Italic
/usr/share/fonts/opentype/urw-base35/URWBookman-Light.otf: URW Bookman:style=Light
/usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf: DejaVu Sans Mono:style=Book
/usr/share/fonts/truetype/dejavu/DejaVuSansCondensed-Oblique.ttf: DejaVu Sans,DejaVu Sans Condensed:style=Condensed Oblique,Oblique
/usr/share/fonts/opentype/urw-base35/NimbusMonoPS-Italic.otf: Nimbus Mono PS:style=Italic
/usr/share/fonts/opentype/urw-base35/P052-Italic.otf: P052:style=Italic
/usr/share/fonts/type1/urw-base35/URWGothic-BookOblique.t1: URW Gothic:style=Book Oblique
/usr/share/fonts/truetype/dejavu/DejaVuSerifCondensed-BoldItalic.ttf: DejaVu Serif,DejaVu Serif Condensed:style=Condensed Bold Italic,Bold Italic
/usr/share/fonts/truetype/dejavu/DejaVuSans-ExtraLight.ttf: DejaVu Sans,DejaVu Sans Light:style=ExtraLight
/usr/share/fonts/type1/gsfonts/n021024l.pfb: Nimbus Roman No9 L:style=Medium Italic
/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf: DejaVu Sans:style=Book
/usr/share/fonts/opentype/urw-base35/NimbusSans-Regular.otf: Nimbus Sans:style=Regular
/usr/share/fonts/type1/urw-base35/URWBookman-DemiItalic.t1: URW Bookman:style=Demi Italic
/usr/share/fonts/opentype/urw-base35/NimbusSansNarrow-Oblique.otf: Nimbus Sans Narrow:style=Oblique
/usr/share/fonts/type1/urw-base35/NimbusSans-Regular.t1: Nimbus Sans:style=Regular
/usr/share/fonts/opentype/urw-base35/C059-Roman.otf: C059:style=Roman
/usr/share/fonts/type1/gsfonts/n021004l.pfb: Nimbus Roman No9 L:style=Medium
/usr/share/fonts/opentype/urw-base35/NimbusMonoPS-BoldItalic.otf: Nimbus Mono PS:style=Bold Italic
/usr/share/fonts/truetype/dejavu/DejaVuSerifCondensed.ttf: DejaVu Serif,DejaVu Serif Condensed:style=Condensed,Book
/usr/share/fonts/truetype/dejavu/DejaVuSerifCondensed-Italic.ttf: DejaVu Serif,DejaVu Serif Condensed:style=Condensed Italic,Italic
/usr/share/fonts/type1/urw-base35/URWGothic-DemiOblique.t1: URW Gothic:style=Demi Oblique
/usr/share/fonts/type1/gsfonts/p052023l.pfb: URW Palladio L:style=Italic
/usr/share/fonts/type1/gsfonts/n022003l.pfb: Nimbus Mono L:style=Regular
/usr/share/fonts/type1/gsfonts/z003034l.pfb: URW Chancery L:style=Medium Italic
/usr/share/fonts/type1/urw-base35/URWBookman-LightItalic.t1: URW Bookman:style=Light Italic
/usr/share/fonts/opentype/urw-base35/NimbusSansNarrow-BoldOblique.otf: Nimbus Sans Narrow:style=Bold Oblique
/usr/share/fonts/type1/gsfonts/c059013l.pfb: Century Schoolbook L:style=Roman
/usr/share/fonts/type1/urw-base35/NimbusSansNarrow-BoldOblique.t1: Nimbus Sans Narrow:style=Bold Oblique
/usr/share/fonts/opentype/urw-base35/NimbusSans-Bold.otf: Nimbus Sans:style=Bold
/usr/share/fonts/type1/urw-base35/NimbusMonoPS-Italic.t1: Nimbus Mono PS:style=Italic
/usr/share/fonts/type1/urw-base35/NimbusRoman-BoldItalic.t1: Nimbus Roman:style=Bold Italic
/usr/share/fonts/opentype/urw-base35/Z003-MediumItalic.otf: Z003:style=Medium Italic
/usr/share/fonts/opentype/urw-base35/NimbusSans-BoldItalic.otf: Nimbus Sans:style=Bold Italic
/usr/share/fonts/opentype/urw-base35/URWGothic-Book.otf: URW Gothic:style=Book
/usr/share/fonts/type1/gsfonts/d050000l.pfb: Dingbats:style=Regular
/usr/share/fonts/truetype/dejavu/DejaVuSerif-BoldItalic.ttf: DejaVu Serif:style=Bold Italic
/usr/share/fonts/type1/gsfonts/n021023l.pfb: Nimbus Roman No9 L:style=Regular Italic
/usr/share/fonts/opentype/urw-base35/D050000L.otf: D050000L:style=Regular
/usr/share/fonts/type1/gsfonts/n019063l.pfb: Nimbus Sans L:style=Regular Condensed Italic
/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Oblique.ttf: DejaVu Sans Mono:style=Oblique
/usr/share/fonts/opentype/urw-base35/C059-BdIta.otf: C059:style=Bold Italic
/usr/share/fonts/truetype/dejavu/DejaVuSerifCondensed-Bold.ttf: DejaVu Serif,DejaVu Serif Condensed:style=Condensed Bold,Bold
/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf: DejaVu Sans:style=Bold
/usr/share/fonts/truetype/dejavu/DejaVuSansMono-BoldOblique.ttf: DejaVu Sans Mono:style=Bold Oblique
/usr/share/fonts/type1/gsfonts/a010013l.pfb: URW Gothic L:style=Book
/usr/share/fonts/opentype/urw-base35/NimbusRoman-Regular.otf: Nimbus Roman:style=Regular
/usr/share/fonts/opentype/urw-base35/C059-Italic.otf: C059:style=Italic
/usr/share/fonts/opentype/urw-base35/NimbusMonoPS-Bold.otf: Nimbus Mono PS:style=Bold
/usr/share/fonts/opentype/urw-base35/NimbusRoman-Bold.otf: Nimbus Roman:style=Bold
/usr/share/fonts/type1/urw-base35/NimbusSansNarrow-Bold.t1: Nimbus Sans Narrow:style=Bold
/usr/share/fonts/opentype/urw-base35/NimbusRoman-BoldItalic.otf: Nimbus Roman:style=Bold Italic
/usr/share/fonts/type1/gsfonts/n019064l.pfb: Nimbus Sans L:style=Bold Condensed Italic
/usr/share/fonts/truetype/dejavu/DejaVuMathTeXGyre.ttf: DejaVu Math TeX Gyre:style=Regular
/usr/share/fonts/truetype/dejavu/DejaVuSansCondensed-BoldOblique.ttf: DejaVu Sans,DejaVu Sans Condensed:style=Condensed Bold Oblique,Bold Oblique
/usr/share/fonts/truetype/droid/DroidSansFallbackFull.ttf: Droid Sans Fallback:style=Regular
/usr/share/fonts/type1/gsfonts/p052004l.pfb: URW Palladio L:style=Bold
/usr/share/fonts/type1/urw-base35/C059-Roman.t1: C059:style=Roman
/usr/share/fonts/truetype/dejavu/DejaVuSansCondensed.ttf: DejaVu Sans,DejaVu Sans Condensed:style=Condensed,Book
/usr/share/fonts/type1/urw-base35/NimbusRoman-Italic.t1: Nimbus Roman:style=Italic
/usr/share/fonts/opentype/urw-base35/NimbusSans-Italic.otf: Nimbus Sans:style=Italic
/usr/share/fonts/type1/gsfonts/n022023l.pfb: Nimbus Mono L:style=Regular Oblique
/usr/share/fonts/type1/urw-base35/URWBookman-Demi.t1: URW Bookman:style=Demi
/usr/share/fonts/type1/gsfonts/a010015l.pfb: URW Gothic L:style=Demi
/usr/share/fonts/truetype/dejavu/DejaVuSansCondensed-Bold.ttf: DejaVu Sans,DejaVu Sans Condensed:style=Condensed Bold,Bold
/usr/share/fonts/type1/urw-base35/URWGothic-Book.t1: URW Gothic:style=Book
/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf: DejaVu Sans Mono:style=Bold
/usr/share/fonts/type1/urw-base35/NimbusRoman-Regular.t1: Nimbus Roman:style=Regular
/usr/share/fonts/opentype/urw-base35/NimbusMonoPS-Regular.otf: Nimbus Mono PS:style=Regular
/usr/share/fonts/opentype/urw-base35/NimbusSansNarrow-Regular.otf: Nimbus Sans Narrow:style=Regular
/usr/share/fonts/type1/gsfonts/n019043l.pfb: Nimbus Sans L:style=Regular Condensed
/usr/share/fonts/truetype/dejavu/DejaVuSans-Oblique.ttf: DejaVu Sans:style=Oblique
/usr/share/fonts/opentype/urw-base35/URWBookman-Demi.otf: URW Bookman:style=Demi
/usr/share/fonts/type1/urw-base35/NimbusMonoPS-Regular.t1: Nimbus Mono PS:style=Regular
/usr/share/fonts/type1/urw-base35/NimbusSans-BoldItalic.t1: Nimbus Sans:style=Bold Italic
/usr/share/fonts/type1/urw-base35/NimbusSans-Bold.t1: Nimbus Sans:style=Bold
/usr/share/fonts/type1/gsfonts/n019044l.pfb: Nimbus Sans L:style=Bold Condensed
/usr/share/fonts/type1/urw-base35/NimbusMonoPS-BoldItalic.t1: Nimbus Mono PS:style=Bold Italic
/usr/share/fonts/type1/urw-base35/StandardSymbolsPS.t1: Standard Symbols PS:style=Regular
/usr/share/fonts/opentype/urw-base35/URWGothic-Demi.otf: URW Gothic:style=Demi
/usr/share/fonts/type1/gsfonts/a010033l.pfb: URW Gothic L:style=Book Oblique
/usr/share/fonts/type1/gsfonts/n021003l.pfb: Nimbus Roman No9 L:style=Regular
/usr/share/fonts/type1/gsfonts/s050000l.pfb: Standard Symbols L:style=Regular
/usr/share/fonts/opentype/urw-base35/NimbusSansNarrow-Bold.otf: Nimbus Sans Narrow:style=Bold
/usr/share/fonts/opentype/urw-base35/URWBookman-DemiItalic.otf: URW Bookman:style=Demi Italic
/usr/share/fonts/opentype/urw-base35/URWGothic-BookOblique.otf: URW Gothic:style=Book Oblique
/usr/share/fonts/type1/gsfonts/b018035l.pfb: URW Bookman L:style=Demi Bold Italic
/usr/share/fonts/type1/gsfonts/c059033l.pfb: Century Schoolbook L:style=Italic
/usr/share/fonts/type1/urw-base35/Z003-MediumItalic.t1: Z003:style=Medium Italic
/usr/share/fonts/type1/urw-base35/P052-BoldItalic.t1: P052:style=Bold Italic
/usr/share/fonts/type1/urw-base35/C059-Italic.t1: C059:style=Italic
/usr/share/fonts/type1/gsfonts/a010035l.pfb: URW Gothic L:style=Demi Oblique
/usr/share/fonts/type1/urw-base35/P052-Roman.t1: P052:style=Roman
/usr/share/fonts/type1/urw-base35/NimbusRoman-Bold.t1: Nimbus Roman:style=Bold
/usr/share/fonts/type1/urw-base35/NimbusSansNarrow-Oblique.t1: Nimbus Sans Narrow:style=Oblique
/usr/share/fonts/opentype/urw-base35/P052-BoldItalic.otf: P052:style=Bold Italic
/usr/share/fonts/truetype/dejavu/DejaVuSerif-Italic.ttf: DejaVu Serif:style=Italic
/usr/share/fonts/type1/urw-base35/D050000L.t1: D050000L:style=Regular
/usr/share/fonts/opentype/urw-base35/StandardSymbolsPS.otf: Standard Symbols PS:style=Regular
/usr/share/fonts/type1/gsfonts/n019023l.pfb: Nimbus Sans L:style=Regular Italic
/usr/share/fonts/type1/gsfonts/b018012l.pfb: URW Bookman L:style=Light
/usr/share/fonts/truetype/noto/NotoMono-Regular.ttf: Noto Mono:style=Regular
/usr/share/fonts/type1/urw-base35/URWGothic-Demi.t1: URW Gothic:style=Demi
/usr/share/fonts/type1/urw-base35/NimbusSans-Italic.t1: Nimbus Sans:style=Italic
/usr/share/fonts/type1/urw-base35/C059-BdIta.t1: C059:style=Bold Italic
/usr/share/fonts/type1/urw-base35/NimbusMonoPS-Bold.t1: Nimbus Mono PS:style=Bold
/usr/share/fonts/type1/gsfonts/c059016l.pfb: Century Schoolbook L:style=Bold
/usr/share/fonts/type1/gsfonts/n022004l.pfb: Nimbus Mono L:style=Bold
/usr/share/fonts/type1/gsfonts/n019024l.pfb: Nimbus Sans L:style=Bold Italic
/usr/share/fonts/type1/gsfonts/b018032l.pfb: URW Bookman L:style=Light Italic
/usr/share/fonts/type1/gsfonts/p052003l.pfb: URW Palladio L:style=Roman
/usr/share/fonts/opentype/urw-base35/URWGothic-DemiOblique.otf: URW Gothic:style=Demi Oblique
/usr/share/fonts/type1/urw-base35/P052-Bold.t1: P052:style=Bold
/usr/share/fonts/type1/gsfonts/n019004l.pfb: Nimbus Sans L:style=Bold
/usr/share/fonts/type1/gsfonts/b018015l.pfb: URW Bookman L:style=Demi Bold
/usr/share/fonts/type1/gsfonts/n022024l.pfb: Nimbus Mono L:style=Bold Oblique
/usr/share/fonts/type1/urw-base35/P052-Italic.t1: P052:style=Italic
/usr/share/fonts/type1/urw-base35/NimbusSansNarrow-Regular.t1: Nimbus Sans Narrow:style=Regular
/usr/share/fonts/type1/urw-base35/C059-Bold.t1: C059:style=Bold
/usr/share/fonts/type1/gsfonts/c059036l.pfb: Century Schoolbook L:style=Bold Italic
/usr/share/fonts/type1/gsfonts/p052024l.pfb: URW Palladio L:style=Bold Italic
/usr/share/fonts/type1/urw-base35/URWBookman-Light.t1: URW Bookman:style=Light
/usr/share/fonts/truetype/dejavu/DejaVuSerif.ttf: DejaVu Serif:style=Book
/usr/share/fonts/opentype/urw-base35/P052-Roman.otf: P052:style=Roman
/usr/share/fonts/opentype/urw-base35/C059-Bold.otf: C059:style=Bold
/usr/share/fonts/opentype/urw-base35/P052-Bold.otf: P052:style=Bold
/usr/share/fonts/opentype/urw-base35/NimbusRoman-Italic.otf: Nimbus Roman:style=Italic
/usr/share/fonts/truetype/dejavu/DejaVuSans-BoldOblique.ttf: DejaVu Sans:style=Bold Oblique
/usr/share/fonts/type1/gsfonts/n019003l.pfb: Nimbus Sans L:style=Regular
```

Based on some searches ....
The fonts referenced in css are not there. So, I did ` sudo apt-get install fonts-roboto`  and installed roboto.
Then I did a `fc-list` and saw roboto in the output.
Did a ` sudo fc-cache -fv`.
Ran DT but the reported behavior did not change.

>BTW are you using darktable-elegant-darker or the default darktable theme?

darktable-icons theme.

**Comment by chhil** [comment]
> And also, do you delete completely destination install folder of darktable AND build folder in folder where you compile darktable, before any new compilation?

Yes.

>you could also show us your CSS/theme settings.
Files from opt/darktable/share/darktable/themes/
[darktable.css.txt](https://github.com/darktable-org/darktable/files/5803465/darktable.css.txt)
[darktable-icons.css.txt](https://github.com/darktable-org/darktable/files/5803466/darktable-icons.css.txt)

**Comment by Mark-64** [comment]
The darktable theme (or darktable-icons) does not use roboto, just the plain sans-serif.
Now that you installed roboto, can you try to select the darktable-elegant-darker theme and see if you still have the issue ?
In the meantime, I will try to understand why none of your fonts is selected

**Comment by Nilvus** [comment]
If you clean completely build and destination folder, so you just have master css. So maybe this is just specific issue about using WSL. Could you test what you have with darktable Windows package. I'm quite sure you will see bold text. Of course, as you use darktable default theme, it depends on OS font, so OS font used need to have bold support of course (most of them have it of course)

Sorry to say (but I need it): Windows is just a piece of shit...

**Comment by chhil** [comment]
@Mark-64 
> Now that you installed roboto, can you try to select the darktable-elegant-darker theme and see if you still have the issue **?**

I get the bold back with this theme. Thanks for the suggestion. Though I don;t like that theme, but will use it until this is resolved for the darktable icons css or there is a workaround that I can use with that theme..

@Nilvus 
> so you just have master css

Don't follow what you are trying to say. I have all the css files that come with the install.

> Could you test what you have with darktable Windows package

Windows  dt I don't have a problem with but I dont use it anymore. WSL has worked fine till now for me.

>Sorry to say that (but I need it): Windows is just a piece of shit...

Its not a windows issue. Its Ubuntu thats installed without a gui so I assume all the fonts are not installed. I run dt through an xserver and it worked fine until now.

**Comment by chhil** [comment]
Update : after changing to darktable-elegant-darker and now changing back to darktable icons, I get the bold css back.

**Comment by Nilvus** [comment]
> Update : after changing to darktable-elegant-darker and now changing back to darktable icons, I get the bold css back.

So, seems to be just a refreshing issue.

**Comment by chhil** [comment]
Every time I start dt, the bold is not there (theme used is darktable icons) . I need to apply the elegant dark them to get it and then change it to darktable icons theme to get the bold back.

**Comment by Mark-64** [comment]
This confirms my guess that is a font selection issue.
Now the problem is that for some reason the plain sans-serif used in the default darktable.css doesn't get selected.
Two options come to my mind:
1) You have no fallback for `font-family: sans-serif`. I see you have Nimbus Sans and DejaVu Sans, I have these in my Ubuntu, but also other sans versions, so maybe you just need to install another sans font which works

2) WSL uses the Windows font backend which doesn not recognize "sans-serif" but onlu "sans". I believe this is unlikely, but it is easy to rule out. Can you try to
- copy darktable.css into my-darktable.css
- in my-darktable.css replace all occurencies of "sans-serif" with "sans"
- in DT, select my-darktable.css as theme
If this doesn't work or screws up the GUI, then we can investigate the first option.

I truly belive DT should come with some (open source) fonts in bundle.

**Comment by chhil** [comment]
I tried the second option and it did not work, there was no bold text.

Then I tried `darktable-icons-darker` this currently works for me after setting it and across restarts.

**Comment by Mark-64** [comment]
> Then I tried `darktable-icons-darker` this currently works for me after setting it and across restarts.

That's logic as `darktable-icons-darker` inherits from `darktable-elegant-darker`

> I tried the second option and it did not work

Sorry if i ask, just to be sure, did you do all the three steps in point 2. ?

**Comment by chhil** [comment]
> Sorry if i ask, just to be sure, did you do all the three steps in point 2. ?

Yes, I followed all the steps.

**Comment by Mark-64** [comment]
> Yes, I followed all the steps.

Ok, this rules out the windows font system, so the fonts must be controlled by fontconfig.
can you give the command `fc-match sans-serif:bold` and post the result?

**Comment by chhil** [comment]
**fc-match sans-serif:bold**
```
DejaVuSans-Bold.ttf: "DejaVu Sans" "Bold"
```



 **fc-match -v sans-serif:bold**
```
Pattern has 38 elts (size 48)
        family: "DejaVu Sans"(s)
        familylang: "en"(s)
        style: "Bold"(s)
        stylelang: "en"(s)
        fullname: "DejaVu Sans Bold"(s)
        fullnamelang: "en"(s)
        slant: 0(i)(s)
        weight: 200(f)(s)
        width: 100(f)(s)
        size: 12(f)(s)
        pixelsize: 12.5(f)(s)
        foundry: "PfEd"(w)
        antialias: True(w)
        hintstyle: 1(i)(w)
        hinting: True(s)
        verticallayout: False(s)
        autohint: False(s)
        globaladvance: True(s)
        file: "/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf"(w)
        index: 0(i)(w)
        outline: True(w)
        scalable: True(w)
        dpi: 75(f)(s)
        scale: 1(f)(s)
        charset:
        0000: 00000000 ffffffff ffffffff 7fffffff 00000000 ffffffff ffffffff ffffffff
        0001: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff
        0002: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff 008873ff
        0003: ffffffff ffffffff f58effff fcff0007 ffffd7f0 fffffffb ffffffff ffffffff
        0004: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff
        0005: ffffffff fffe003f fe7fffff fffffffe 000006ff ffff0000 ffff00cf 001f07ff
        0006: 882016c0 07fffffe 04bfffff fe11ffff ffffffff ffffffff 002159c0 03ff0000
        0007: 00000000 00000000 00000000 00000000 00000000 00000000 ffffffff 073ff8ff
        000e: 00000000 80000000 00000000 00000000 fef02596 3bffecae 33ff3f5f 00000000
        0010: 00000000 00000000 00000000 00000000 00000000 ffffffff ffff003f 1fffffff
        0014: effffefe ffbfffff fff7f7ff ffffffff ffffffff 3fffffff ffffffff fffff7ff
        0015: ffff00ff 7fffffff fffdffff fff007ff 007ffc3f 0000ffff 40000000 00000002
        0016: 00000000 00000000 000000c0 007fc000 1fffffff 00000000 00000000 00000000
        001d: ffdfffff ffff7fcf efffffff 298007ff f8000020 ffffffff 000003f0 00000000
        001e: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff 0fffffff
        001f: 3f3fffff ffffffff aaff3f3f 3fffffff ffffffff ffdfffff efcfffdf 7fdcffff
        0020: ffffffff ffffffff ffffffff fff3fc1f 1fff7fff 273fffff 18c30000 00000002
        0021: fffffbff ffffffff ffff4bff ffffffff ffff023f ffffffff ffffffff ffffffff
        0022: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff
        0023: f303ffff 000019f3 00000000 24380000 f8100080 00007fff 0000c000 00000128
        0024: 00000000 0000000c 00000000 000003ff 00000000 00000000 00000000 00000000
        0025: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff
        0026: ffffffff ffffffff ffffffff ffffffff dfffffff 01ffffff 0000000f 00000004
        0027: fffff3de fffffeff 7f47afff fffffffe ff1fffff 7ffeffff 00000060 ffff0fc1
        0028: ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff ffffffff
        0029: 00000cc0 00000000 00000003 00000000 00000018 00000000 003fc000 0c000800
        002a: 1ffff007 00008000 00000000 e0000c00 ffffffff 07ffc001 00000000 06000000
        002b: 87ffffff 0000001f 00180000 00000000 00000000 00000000 00000000 00000000
        002c: 00000000 00000000 00000000 feffffff 00000000 00000000 00000000 00000000
        002d: ffffffff ffff003f ffffffff 0000803f 00000000 00000000 00000000 00000000
        002e: 81000000 0000403c 00000000 00000000 00000000 00000000 00000000 00000000
        004d: 00000000 00000000 00000000 00000000 00000000 00000000 ffffffff ffffffff
        00a4: 00000000 00000000 00000000 00000000 00000000 00000000 ffff0000 ffffffff
        00a6: 00000000 00000000 00f330f0 00007ffc 03303c00 00000000 00000000 00000000
        00a7: f87fff00 ffff0ffc 00cfcfc3 000000f0 00037e0f 000007ff 00000000 ff000000
        00ef: 03ffffff 00000000 00000000 00000000 00000000 00000000 00000000 00000000
        00f4: ffffffff ffffff7f 00000003 00000000 00000000 00000000 00000000 00000000
        00f6: 00000000 00000000 00000000 00000000 00000000 00000000 00000020 00000000
        00fb: e0f8007f 5f7fffff fffcffdb ffffffff ffffffff 00003c0f dff80000 f00003f0
        00fe: 0000ffff 0000000f 00000000 ffdf0000 ffffffff ffffffff ffffffff 9fffffff
        00ff: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 3e000000
        0103: 7fffffff 0000000f 00000000 00000000 00000000 00000000 00000000 00000000
        01d3: ffffffff ffffffff 007fffff 00000000 00000000 00000000 00000000 00000000
        01d5: 00000000 7b000000 fffdfc5f 00000fff 00000000 00000000 fff00000 ffffffff
        01d6: 000000ff 00000000 00000000 00000000 00000000 00000000 00000000 00000000
        01d7: 00000000 00000000 ffc00000 ffffffff 0000ffff 00000000 ff000000 003ff003
        01f0: 00000000 ffff0000 ffffffff ffffffff 000fffff 7ffe7fff fffefffe 00000000
        01f3: 01fe0000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
        01f4: 00000000 00226000 00000000 00000000 00000000 00000000 00000000 00000000
        01f6: ffffffff ffffefef 00000009 00000000 00000000 00000000 00000000 00000000
(w)
        lang: aa|ab|af|ar|ast|av|ay|az-az|az-ir|ba|bm|be|bg|bi|bin|br|bs|bua|ca|ce|ch|chm|co|cs|cu|cv|cy|da|de|el|en|eo|es|et|eu|fa|fi|fj|fo|fr|ff|fur|fy|ga|gd|gl|gn|gv|ha|haw|he|ho|hr|hu|hy|ia|ig|id|ie|ik|io|is|it|iu|ka|kaa|ki|kk|kl|ku-am|ku-ir|kum|kv|kw|ky|la|lb|lez|ln|lo|lt|lv|mg|mh|mi|mk|mo|mt|nb|nds|nl|nn|no|nr|nso|ny|oc|om|os|pl|pt|rm|ro|ru|sah|sco|se|sel|sh|shs|sk|sl|sm|sma|smj|smn|sms|so|sq|sr|ss|st|sv|sw|tg|tk|tl|tn|to|tr|ts|tt|tw|tyv|ug|uk|uz|ve|vi|vo|vot|wa|wen|wo|xh|yap|yi|yo|zu|ak|an|ber-dz|ber-ma|crh|csb|ee|fat|fil|hsb|ht|hz|jv|kab|kj|kr|ku-iq|ku-tr|kwm|lg|li|mn-mn|ms|na|ng|nv|ota|pap-an|pap-aw|qu|quz|rn|rw|sc|sd|sg|sn|su|ty|za(s)
        fontversion: 155320(i)(s)
        capability: "otlayout:DFLT otlayout:arab otlayout:armn otlayout:brai otlayout:cans otlayout:cher otlayout:cyrl otlayout:geor otlayout:grek otlayout:hani otlayout:hebr otlayout:kana otlayout:lao  otlayout:latn otlayout:math otlayout:nko  otlayout:ogam otlayout:runr otlayout:tfng otlayout:thai"(w)
        fontformat: "TrueType"(w)
        embeddedbitmap: True(s)
        decorative: False(s)
        lcdfilter: 1(i)(w)
        namelang: "en"(s)
        prgname: "fc-match"(s)
        postscriptname: "DejaVuSans-Bold"(w)
        color: False(w)
        symbol: False(s)
        variable: False(s)
```

**Comment by Mark-64** [comment]
Ok, I found the issue.
With PR #7671 the control of bold menu items has been moved to CSS, however the statement

```
.active-menu-item *
{
    font-weight: bold;
}

is placed in darktable-elegant-darker.css, and it should be in darktable.css instead.
I will fix this with another commit to #7835
```

**Comment by Nilvus** [comment]
> is placed in darktable-elegant-darker.css, and it should be in darktable.css instead.
> I will fix this with another commit to #7835

Didn't notice that too before. Indeed, should be in darktable.css. Good catch!

**Comment by mchhil** [comment]
I hanged my install css files to mimic the commit.
The css fix works for me with the darktabe.css as my theme.
